src/core/public/notifications/toasts/toasts_api.tsx

```diff
@@ -66,14 +93,22 @@ export class ToastsApi {
     this.uiSettings = deps.uiSettings;
   }
 
+  /** @internal */
   public registerOverlays(overlays: OverlayStart) {
     this.overlays = overlays;
   }
 
+  /** Current array of toast messages to show to user. */
```

Comment:
This returns an observable, should the message here change to not say `array`?

Reply:
I will address this in the original PR: #47069